src/classes/GS_NonprofitTrialOrgService.cls

```diff
@@ -161,9 +162,11 @@ public with sharing class GS_NonprofitTrialOrgService {
      * @return Organization with TrialExpirationDate and IsSandbox queried.
      */
     private static Organization getOrgInformation() {
-        return [SELECT TrialExpirationDate,
-                       IsSandbox
-                FROM   Organization];
+        return [
+            SELECT TrialExpirationDate, IsSandbox
+            FROM   Organization
+            WITH SECURITY_ENFORCED
```

Comment:
This clause should only be used on standard and custom objects, not system objects.